src/components/tasks/card/index.tsx

```diff
@@ -38,8 +34,15 @@ const Card: FC<Props> = ({ content }) => {
   const iconHeight = '25px';
   const iconWidth = '25px';
 
+  const cardClassNames = [classNames.card];
+  const currentDate = new Date();
+  if (status !== COMPLETED) {
+    if (localEndsOn.valueOf() - currentDate.valueOf() <= 0) {
+      cardClassNames.push(classNames.overdueTask);
+    }
+  }
   return (
-    <div className={classNames.card}>
+    <div className={cardClassNames.join(' ')}>
```

Comment:
Once you define `isTaskOverdue` function, use it here for conditional class names using template literals.

```HTML
<div className={`
    ${classNames.card}
    ${isTaskOverdue() && classNames.overdueTask}
`}>
.
.
</div>
```

We'll get someone to write a utility function for this kind of case in the future.

Reply:
Done 😁